components/hacbs/integration-service/base/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+resources:
+- "https://github.com/redhat-appstudio/integration-service/config/kcp?ref=cf8459fbde3cfa5f92518e637fb55150500b8626"
```

Comment:
https://github.com/redhat-appstudio/integration-service/blob/main/config/kcp/kustomization.yaml#L21 has another apiexport that isn't really needed anymore since there is an apiexport for each overlay as part of this PR. You will want to turn that one off in the integration-service repo.

Michal did some magic for application-service that I copied for release-service if you want to do that instead. See https://github.com/redhat-appstudio/application-service/pull/192/files and https://github.com/redhat-appstudio/infra-deployments/pull/790/files

Reply:
Ok, let me update them in https://github.com/redhat-appstudio/integration-service/pull/53. Thanks!